custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go

```diff
@@ -227,7 +227,8 @@ func (t *Translator) metricsFor(values map[string]resource.Quantity, groupResour
 
 	for _, item := range podList.Items {
 		if _, ok := values[fmt.Sprintf("%s", item.GetUID())]; !ok {
-			return nil, provider.NewMetricNotFoundForError(groupResource, metricName, item.GetName())
+			glog.V(4).Infof("Metric not '%s' not found for pod '%s'", metricName, item.Name)
```

Comment:
Something wrong with this log message. I think you have extra "not" before metric name.

Reply:
Right, fixed